build.sh

```diff
@@ -86,3 +86,10 @@ function build_codemirror {
 }
 
 run $BUILD/CodeMirror build_codemirror
+
+# Build pegjs parsers
+function build_pegjs_parsers {
+    bin/pegjs --export-var haddock_parser --format globals -o ../haddock-parser.js ../../web/js/haddock-parser.pegjs
```

Comment:
Does `haddock-parser.pegjs` need to be accessible at runtime?  If not, then it shouldn't go in `web`.

Reply:
Ok, what is better place for it?

Reply:
Moved to root of project.